eng/PublishRelease.ps1

```diff
@@ -21,6 +21,8 @@ try
    Write-Host "Publishing $file"
 
    cmd /c "npx -q publish-release --token $Token --repo autorest.csharp --owner azure --name $name --tag $devVersion --notes=prerelease-build --prerelease --editRelease false --assets $file --target_commitish $Sha 2>&1"
+
+   Write-Host "##vso[task.setvariable variable=AutorestCSharpVersion;isSecret=false]https://github.com/Azure/autorest.csharp/releases/download/$devVersion/autorest-csharp-v3-$devVersion.tgz"
```

Comment:
Should we reduce this to just the version number?

Reply:
For now I need to pass the entire path to autorest because we run from github releases. I tried to figure out how to use version only but didn't find an easy way yet. Maybe next time.

Reply:
Can we not just pass the version and format the github release stuff around the version in the targets file?

Reply:
What does that improve? At least this way all the logic is part of this repo.

Reply:
I guess nothing really beyond the PRTitle being a little long and scary, but it looks like you already removed the version from that so maybe nothing to see here :)